x/auth/legacy/legacytx/stdtx_builder.go

```diff
@@ -96,7 +96,7 @@ func (s StdTxConfig) TxEncoder() sdk.TxEncoder {
 }
 
 func (s StdTxConfig) TxDecoder() sdk.TxDecoder {
-	return DefaultTxDecoder(s.Cdc)
+	return mkDecoder(s.Cdc.UnmarshalBinaryBare)
```

Comment:
What's the justification for refactoring this?

Reply:
removing code duplication.